timemachine/parallel/utils.py

```diff
@@ -43,3 +43,7 @@ def batch_list(values: List, num_workers: int) -> List[List]:
     for i, value in enumerate(values):
         batched_values[i % num_workers].append(value)
     return list(batched_values.values())
+
+
+def flatten_list(results: List[List[Any]]) -> List[Any]:
```

Comment:
nit: could use a `TypeVar` here for a more precise type (otherwise e.g. `foo: List[str] = flatten_list([[1, 2], [3]])` would typecheck)
```suggestion
A = TypeVar("A")
def flatten_list(results: List[List[A]]) -> List[A]:
```

Reply:
inlined in run_smc